### PR TITLE
New/Update Rules

### PR DESCRIPTION
**Pull request by nasbench**
- Created rule for "Forfiles" usage as a LOLBIN
- Created rule for "Pcalua" usage as a LOLBIN
- Created rule for using "Conhost" with path traversal
- Update "Explorer" Process Tree Break" rule. As it doesn't actually require the "/root" flag. It can launch with or without the flag but the new process will spawn with the "/factory,{75dff2b7-6936-4c06-a8bb-676a7b00b24b}" command line and that's what the new logic implies.

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_conhost_path_traversal.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+title: Conhost.exe CommandLine Path Traversal
+id: ee5e119b-1f75-4b34-add8-3be976961e39
+status: experimental
+description: detects the usage of path traversal in conhost.exe indicating possible command/argument confusion/hijacking
+author: Nasreddine Bencherchali
+references:
+  - https://pentestlab.blog/2020/07/06/indirect-command-execution/
+date: 2022/06/14
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_conhost_path_traversal.yml:8]: `- https://hackingiscool.pl/cmdhijack-command-argument-confusion-with-path-traversal-in-cmd-exe/`
find in the first ref.
Test on windows 11, works well 🤣

File: rules/windows/process_creation/proc_creation_win_lolbin_forfiles.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    selection_img:
+        - Image|endswith: \forfiles.exe
+        - OriginalFileName: forfiles.exe
+    selection_cli:
+        CommandLine|contains|all:
+            # The CLI accepts combination of both "slash" and "dash"
+            - ' /p '
+            - ' /m '
+            - ' /c '
+    condition: all of selection*
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_lolbin_forfiles.yml:23]:
```yaml
    selection_cli_p:
        CommandLine|contains:
            - ' /p '
            - ' -p '
    selection_cli_m:
        CommandLine|contains:
            - ' /m '
            - ' -m '
     selection_cli_c:
        CommandLine|contains:
            - ' /c '
            - ' -c ' 
```
Get all the permutation

Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_lolbin_forfiles.yml:29]: Sometimes my brain shuts down and I can't even think of stuff like this 😂 Thanks

File: rules/windows/process_creation/proc_creation_win_susp_explorer_break_proctree.yml [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
-description: Detects a command line process that uses explorer.exe /root, which is similar to cmd.exe /c, only it breaks the process tree and makes its parent a new instance of explorer
-author: Florian Roth
+description: Detects a command line process that uses explorer.exe to launch arbitrary commands or binaries, which is similar to cmd.exe /c, only it breaks the process tree and makes its parent a new instance of explorer spawning from "svchost"
+author: Florian Roth, Nasreddine Bencherchali
 references:
-  - https://twitter.com/CyberRaiju/status/1273597319322058752
-  - https://twitter.com/bohops/status/1276357235954909188?s=12
+    - https://twitter.com/CyberRaiju/status/1273597319322058752
+    - https://twitter.com/bohops/status/1276357235954909188?s=12
+    - https://twitter.com/nas_bench/status/1535322450858233858
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_susp_explorer_break_proctree.yml:9]: `- https://securityboulevard.com/2019/09/deobfuscating-ostap-trickbots-34000-line-javascript-downloader/`
not find a better ref to explain

**Review by frack113 — APPROVED** [review]

**Pull request merged**